src/windows.rs

```diff
@@ -28,3 +29,24 @@ pub fn kernel_version() -> Option<String> {
         }
     }
 }
+
+/// Get the build number from Windows.
+///
+/// **Note**: On Windows 8 and later this will report the Windows 8 build number 9200,
+/// unless the final application is build to explicitly target Windows 10.
+/// See [`GetVersionEx`] for details.
+///
+/// [`GetVersionEx`]: https://docs.microsoft.com/en-us/windows/win32/api/sysinfoapi/nf-sysinfoapi-getversionexa
+///
+///
+/// Returns `None` if an error occured.
+pub fn windows_build_number() -> Option<i32> {
```

Comment:
CI fails because this is never actually exposed anywhere.
That's because [here we only re-export a single function](https://github.com/badboy/whatsys/blob/main/src/lib.rs#L50).
This needs a

```rust
#[cfg(target_os = "windows")]
pub use system::windows_build_number;
``` 

in `src/lib.rs`.

Reply:
👍 ah ok, I had expected that to be covered by the prior `use system` within the platform bit, updated!

Reply:
Nah, `use system` is using it only "privately", we would have to use `pub use system` to export it, but we don't want to export the whole module.